mvpa2/tests/test_transerror.py

```diff
@@ -673,6 +673,26 @@ def test_confusionmatrix_nulldist(self):
                                       cvnp[(np.array([0,1]), np.array([1,0]))])
 
 
+def test_plotting_subset():
+    """ smoketest to see whether plotting only a subset of existing labels blows"""
+
+    from mvpa2.misc.data_generators import normal_feature_dataset
+    from mvpa2.clfs.gnb import GNB
+    ds = normal_feature_dataset(nlabels=4)
+    clf = GNB()
+    cv = CrossValidation(
+        clf, NFoldPartitioner(),
+        errorfx=None,
+        enable_ca=['stats'])
+    res = cv(ds)
+    # get all labels
+    labels = cv.ca.stats.labels
+    cv.ca.stats.plot(labels=labels, numbers=True)
```

Comment:
for unit testing `ConfusionMatrix` there is no need to do full blown classification.  Just create an instance of ConfusionMatrix (see other tests) and work with that.